src/peergos/shared/user/CommittedWriterData.java

```diff
@@ -0,0 +1,16 @@
+package peergos.shared.user;
+
+import peergos.shared.merklebtree.*;
+
+public class CommittedWriterData {
```

Comment:
I think the name ``WriterDataWithHash`` would be clearer here?  I mean, there nothing  to stop me  from creating  an  instance where I  hadn't committed  the ``WriterData``.